src/replication.thrift

```diff
@@ -478,6 +478,28 @@ struct query_disk_info_response
     4:list<disk_info> disk_infos;
 }
 
+// This request is sent from client to replica_server.
+struct replica_disk_migrate_request
+{
+    1:dsn.gpid pid
+    2:string origin_disk;
+    3:string target_disk;
```

Comment:
```suggestion
    2:string origin_disk;
    3:string target_disk;
```

Add an example here. Is “disk” a full path?

Reply:
disk_tag, such `ssd1`, I will add it